website/src/pages/about/index.tsx

```diff
@@ -0,0 +1,136 @@
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
```

Comment:
Hello, I'm unsure core values should land in about/ page

Reply:
any suggestions on how to present this content? @benoitf @slemeur

Reply:
probably name it with what the file provides. 'core-values' , 'values' or 'mission' , 'mission-statement', etc.

Reply:
Addressed with a new "core-values" folder added.